src/test/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/producer/MessageProducerImplUnitTest.java

```diff
@@ -56,11 +62,15 @@
     @Captor
     private ArgumentCaptor<Message> kafkaMessageCaptor;
 
+    @BeforeEach
+    void setup() {
+        when(serializerFactory.getSpecificRecordSerializer(ChipsRestInterfacesSend.class)).thenReturn(avroSerializer);
+    }
+
```

Comment:
can you add the blank lines back in between the method call and assertions, they make the test easier to read

Reply:
Alright then.